### DISCUSSION TITLE
Getting NPM error code 128 when running Vercel CLI

### DISCUSSION THREAD

**Discussion by DragonOsman** [vercel/vercel#11842] [Help]
### Question

I tried to deploy my Node backend code to Vercel by running the `vercel` command, but got NPM error code 128.  Full message:
```
Error: npm ERR! code 128
npm ERR! Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
npm ERR! git@github.com: Permission denied (publickey).
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
```
The [repository](https://github.com/DragonOsman/personal-library) exists and I do have access rights (and of course, it's a public repository), but how do I verify that on Vercel?  Is there a way?

Also, how can I make sure that my Node backend has the url https://personal-library-server.vercel.app, and that the client side has https://personal-library-client.vercel.app?  Is it possible to do this?

**Comment by GoreStarry** [comment]
I have the same issue when I create a `new` project on Vercel, but it still works on all my old projects.
That's happened when I install a private repo by
```
"package-name": "git+https://<username>:<github_token>@github.com/<user>/<repo>.git"
```
All of my old projects use the same token to install the same package and are still fine even if redeployed.
